app/presenters/two_factor_authentication/set_up_piv_cac_selection_presenter.rb

```diff
@@ -19,7 +19,8 @@ def phishing_resistant?
     end
 
     def recommended?
-      user.confirmed_email_addresses.any?(&:gov_or_mil?)
+      # Not sure yet if we want to change the recommended for this since that list only does .gov not mil emails
```

Comment:
I don't think we want to drill down to that level because some state and local government agencies issue [PIV-I cards](https://www.idmanagement.gov/university/pivi/#2-minimum-credential-requirements)

Reply:
More to `.mil`, the question I'm wondering is if we can reliably expect that anyone with a `.mil` would have a [CAC card](https://www.cac.mil/common-access-card/).

Reply:
Filtering `.mil` emails should be okay.

Reply:
Yea I think .mil cause the https://raw.githubusercontent.com/cisagov/dotgov-data/main/current-full.csv does not contain any military domains.